docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,217 @@
+As a result, the browsers will be connected to TestCafe and tests will start.
+
+## File Path/Glob Pattern
+
+The `file-or-glob ...` argument specifies files or directories(separated by a space) to run tests from.
+
+For example, this command runs all tests located in the `/my-tests` directory.
+
+```sh
+testcafe ie /my-tests
+```
+
+The following command runs tests from the certain fixture files.
+
+```sh
+testcafe ie /tests/fixture1.js /tests/fixture2.js
+```
+
+You can also use [globbing patterns](https://github.com/isaacs/node-glob#glob-primer) to specify a set of files. 
+For example, this command runs tests from files that matches `/tests/*page*`, namely `/tests/example-page.js` and `/tests/main-page.js`.
+
+```sh
+testcafe ie /tests/*page*
+```
+
+If you do not specify any file or directory, TestCafe will run tests from `/test` or `/tests` directories.
+
+## Options
+
+### -h, --help  
+Displays usage information for commands.
+
+```sh
+testcafe --help
+```
+
+### <a class="anchor" name="b"></a>-b, --list-browsers 
+Lists [aliases] of the browsers available on this computer.
+
+```sh
+testcafe --list-browsers
+```
+
+### -r \<name\>, --reporter \<name\>  
+Specifies a built-in or custom reporter that will be used to generate test reports. 
+The available built-in reporters are `json`, `list`, `minimal`, `spec`, `xunit`. By default, the `spec` reporter is used.
+
+The following command tells TestCafe to run the tests in all available browsers and generate a report in the xunit format.
+
+```sh
+testcafe all /tests/sample-fixture.js -r xunit
+```
+
+The following command tells TestCafe generate a test report by using the custom plugin reporter.
+
+```sh
+testcafe all /tests/sample-fixture.js -r my-reporter
+```
+
+A generated test report will be displayed in the command prompt window. 
+
+If you need to save a report to an external file, you can redirect the command output stream to a file  
+by using the > redirection operator and specifying a report file path.
+
+```sh
+testcafe all /tests/sample-fixture.js > /tests/test-results.txt
+```
+
+### -s \<path\>, --screenshots \<path\> 
+Enables screenshot capturing and specifies the directory path
+to save the screenshots to.
+
+```sh
+testcafe all /tests/sample-fixture.js -s /screenshots
+```
+
+### -S, --screenshots-on-fails
+Takes a screenshot whenever a test fails. Screenshots are saved to the directory
+specified by using the **-screenshots \<path\>** option.
+
+For example, the following command tells TestCafe to run tests from the
+  *sample-fixture.js* file in all browsers, take screenshots if tests fail
+  and save the screenshots to the */screenshots* directory.
+
+```sh
+testcafe all /tests/sample-fixture.js -S -s /screenshots
+```
+
+### -q, --quarantine-mode 
+Enables the quarantine mode for tests that fail. In this mode,
+a failed test runs five more times, and if it fails in all attempts,
+the test is marked as failed. If the test passes in any run,
+it is marked as unstable.
```

Comment:
Seems like it slightly differs from what we have in 
https://github.com/DevExpress/testcafe/pull/298/files#diff-679a3d497074db631b0e722a5db01a5eR238

Particularly, the point about marking tests failed if they fail in **all** attemts